consensus/spos/sposFactory/constants.go

```diff
@@ -1,3 +1,4 @@
 package sposFactory
 
 const blsConsensusType = "bls"
+const maxDelayCacheSize = 1
```

Comment:
why so small?

Reply:
Wanted to force previous data (if any) to be broadcast as soon as possible